custom_components/places/helpers.py

```
@@ -43,7 +56,13 @@ def get_dict_from_json_file(name: str, filename: str, json_folder: str) -> Mutab
 
 
 def remove_json_file(name: Any, filename: Any, json_folder: Any) -> None:
```

Ruff (0.15.12)

[warning] Dynamically typed expressions (typing.Any) are disallowed in `name`

(ANN401)